nixos/modules/services/web-apps/gancio.nix

```diff
@@ -222,8 +225,19 @@ in
           ${concatMapStringsSep "\n" (p: "ln -sf ${p} plugins/") cfg.plugins}
         '';
 
+        # Set permission so that nginx can use gancio socket
+        # (gancio/nuxt does not support systemd socket activation)
+        postStart = ''
+          until [ -S ${cfg.settings.server.socket} ]; do
+            echo Waiting for ${cfg.settings.server.socket}
+            sleep 1
+          done
+          chmod 0660 ${cfg.settings.server.socket}
```

Comment:
Just curious, does this mean there's a small time window where the socket might be `0666`? Has there been any discussion upstream about setting socket permissions using their own configuration? Or maybe it would be possible for us to do it through systemd's `UMask` option (unless it messes up any other permissions)?

Reply:
The socket is created with `0600` permission. I guess using `UMask` would make other files a little more accessible than necessary, but probably not a big deal when using postgres as database. And it might be necessary anyway when configuring nginx for caching gancio static resources.
I'll try.

Reply:
Also, from a quick search in Nuxt (the web framework used by gancio) github issues I did not see any mention of offering permission configuration for the unix socket. I don't have time to start such a feature request right now.

Reply:
Used `UMask = "0002"` in last update.

Reply:
Okay, going from 0600 to 0660 isn't as much ground for concern. Consider adding a small comment